tests/testsuite/cargo_add/rust_version_older/mod.rs

```diff
@@ -0,0 +1,35 @@
+use cargo_test_support::compare::assert_ui;
+use cargo_test_support::prelude::*;
+use cargo_test_support::Project;
+
+use crate::cargo_add::init_registry;
+use cargo_test_support::curr_dir;
+
+#[cargo_test(nightly, reason = "-Zmsrv-policy is unstable")]
```

Comment:
Why are we filtering these tests for nightly?

Reply:
ah I misunderstood what `#[cargo_test(nightly)]` was actually for (*external* nightly features), will fix